### PR TITLE
Live radio just audio

### PR DESCRIPTION
**Pull request by Kszesiek**
Live radio streaming using Just Audio and Audio Service libraries. Already merged with all the latest changes on master. Left some unused code as comments for potential help with future development. Also added an icon to the app and app notifications.

### CODE REVIEW

File: mobile/lib/state/audio_handler_cubit.dart [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        androidNotificationOngoing: true,
+      ),
+    );
+    // setting proper AudioService object as state, notifying listeners
+    emit(audioService);
+  }
+
+  AudioHandler? get() {
+    return state;
+  }
```
Comment (julianuziemblo) [inline mobile/lib/state/audio_handler_cubit.dart:29]: Unnecessary - there already is a public getter, can delete (if it's not intended to be further modified or overriden - if so, needs a clarifying comment)
```suggestion
```

**Review by julianuziemblo — COMMENTED** [review]
Overall: great job, everything works fine and as it should (although, as a "fun fact" - the WEITI wifi seems to prevent the radio from working). I just left a few comments regarding (maybe subjective...) code cleanliness. One issue I noticed is that the stream title doesn't seem to be working correctly... I tested it on multiple different internet connections and on multiple times of day, but I can't get it to work, the only thing I get is the placeholder 'Radio Aktywne' (as in the picture below).
Stream title not working

File: mobile/lib/components/radio_player/audio_player_handler.dart [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+      }
+      if (streamTitle.isNotEmpty) {
+        mediaItemValue = mediaItemValue.copyWith(
+          title: streamTitle,
+          // title: streamTitle.isNotEmpty
+          //     ? streamTitle
+          //     : 'Stream title not provided',
+        );
+      }
+      mediaItem.add(mediaItemValue);
```
[the remark this answers is not in the document]
Comment (Kszesiek) [reply mobile/lib/components/radio_player/audio_player_handler.dart:51]: Fair point 👍

**Review by julianuziemblo — APPROVED** [review]
Everything works fine (even the stream title part finally), so - approved!

obraz

**Review by julianuziemblo — APPROVED** [review]
Fixed all conflicts, good to go

**Pull request merged**